changelog/3770.feature.rst

```diff
@@ -0,0 +1 @@
+Fido clients (subclasses of `sunpy.net.base_client.BaseClient`) can now register their own attrs modules with `sunpy.net.attrs`. This allows clients which require attr classes specific to that client to register modules that can be used by the user i.e. ``a.vso``. It also allows clients implemented externally to sunpy to register attrs.
```

Comment:
```suggestion
Fido clients (subclasses of `sunpy.net.base_client.BaseClient`) can now register their own attrs modules with `sunpy.net.attrs`. This allows clients which require attr classes specific to that client to register attributes that can be used by the user i.e. ``a.vso``. It also allows clients implemented externally to sunpy to register attrs.
```
... unless I'm reading something wrong?

Reply:
You register a module (i.e. `a.vso` maps to `sunpy.net.vso.attrs`) which contains attrs (but is just a module). [here](https://github.com/sunpy/sunpy/pull/3869/files#diff-85c0984f67d7abb954d0f4a8d56fcb4fR46)